packages/Alert/src/index.tsx

```diff
@@ -1,15 +1,15 @@
 import React, { Children, cloneElement } from 'react'
-import { Stack } from '@welcome-ui/stack'
 import { Box } from '@welcome-ui/box'
 import { CloseButton } from '@welcome-ui/close-button'
 import { Button, ButtonProps } from '@welcome-ui/button'
 import { CreateWuiProps, forwardRef } from '@welcome-ui/system'
-import { Variant, VariantIcon } from '@welcome-ui/variant-icon'
+import { VariantIcon } from '@welcome-ui/variant-icon'
 
 import * as S from './styles'
 import { Title } from './Title'
 
 export type Size = 'sm' | 'md'
+export type Variant = 'danger' | 'success' | 'warning' | 'info' | 'default' | 'beige'
```

Comment:
Do we still use `'info'` ?

Reply:
Only on Alert / Growl / Snarkbar / Tag :)